query-engine/core/src/executor/execute_operation.rs

```diff
@@ -259,6 +269,7 @@ async fn execute_on<'a>(
     query_schema: &'a QuerySchema,
     trace_id: Option<String>,
 ) -> crate::Result<ResponseData> {
+    #[cfg(feature = "metrics")]
```

Comment:
Why aren't the metrics macros shimmed as the tracing macros where shimmed in query-connector?

Reply:
For convenience: in `sql-query-connector`, the `trace!` macro was used twice in a single file (`src/database/operations/write.rs`) among the files included for the `wasm32` compilation.

That shim is effectively a shortcut to avoid adding a currently un-needed `metrics` feature flag in `sql-query-connector`.